e2e/full_test.go

```diff
@@ -0,0 +1,68 @@
+package e2e
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/celestiaorg/orchestrator-relayer/evm"
+	"github.com/celestiaorg/orchestrator-relayer/rpc"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// TestFullLongBehaviour mainly lets a multiple validator network run for 200 blocks, then checks if
+// the valsets and data commitments are relayed correctly.
+func TestFullLongBehaviour(t *testing.T) {
```

Comment:
the name "FullLong" is not very precise as to what is actually being tested, so I would definitely recommend using different name. 

Simply TestIntegrationTest or TestIntegrationSuite or anything standard

Reply:
What is does is run the orchestrator and relayer for  more than 200 blocks. It's a long test that runs the full behaviour :D 

`TestIntegrationTest` is too generic, at least the other name is stating `FullLong`... I will open an issue for it https://github.com/celestiaorg/orchestrator-relayer/issues/206